src/mango/src/mango_cursor_view.erl

```diff
@@ -427,7 +435,10 @@ doc_member(Cursor, RowProps) ->
                     match_doc(Selector, Doc, ExecutionStats1);
                 Else ->
                     Else
-            end
+            end;
+        null ->
```

Comment:
What about instead just returning `{no_match, null, {execution_stats, ExecutionStats1}}` here. Then we don't need to add in the `{ok, null, Stats}` clause in the `handle_message` function? It seems unnessecary to handle this cause separately everywhere. Rather handle it in one place and simplify it down to a `no_match`.

Reply:
refactored according this good suggestion